nidata/multimodal/hcp/datasets.py

```diff
@@ -51,6 +52,12 @@ def __init__(self, data_dir=None, fetcher_type='http', profile_name=None, access
                                           passwd=passwd)
         else:
             raise NotImplementedError(fetcher_type)
+    def prepend(self, src_files, files):
```

Comment:
Rather than pass `files` and make changes into that variable (that's called "side-effects", and it's generally frowned upon), better not to pass `files` at all. Instead, make a local variable `files`, and return it.